### PR TITLE
Clean up errored assets

### PR DESCRIPTION
**Pull request by pereirinha**
## Approach

- Adds means to clean up the sync error flag, allowing the assets to be retried to sync. This can be achieved by one of these:
  - Via the dashboard;
  - Via the Media Library;
  - Via the CLI commands

### CODE REVIEW

File: php/sync/class-upload-sync.php [hunk 1]
```diff
@@ -91,7 +90,8 @@ private function register_hooks() {
 			'bulk_actions-upload',
 			function ( $actions ) {
 				$cloudinary_actions = array(
-					'cloudinary-push' => __( 'Sync with Cloudinary', 'cloudinary' ),
+					'cloudinary-push'     => __( 'Sync with Cloudinary', 'cloudinary' ),
+					'cloudinary-clean-up' => __( 'Clean Cloudinary error', 'cloudinary' ),
```
Comment (tharsheblows) [inline php/sync/class-upload-sync.php:94]: **🥜 nitpick**
This is so small but I think "error" should be capitalised to keep it consistent?

File: php/sync/class-upload-sync.php [hunk 2]
```diff
@@ -136,18 +136,35 @@ public function add_inline_action( $actions, $post ) {
...
+				add_query_arg(
+					array(
+						'action'   => 'cloudinary-clean-up',
+						'media[]'  => $post->ID,
+						'_wpnonce' => wp_create_nonce( 'bulk-media' ),
+					),
+					admin_url( 'upload.php' )
+				),
+				esc_attr__( 'Clean Cloudinary error', 'cloudinary' ),
+				esc_html__( 'Clean Cloudinary error', 'cloudinary' )
```
Comment (tharsheblows) [inline php/sync/class-upload-sync.php:164]: **🥜 nitpick**
Same here, maybe capitalise Error to keep it consistent

**Review by tharsheblows — APPROVED** [review]
There are two very small comments regarding capitalisation. If you think they don't matter then you can ignore!

I've tested all three scenarios also and they are working.

File: php/sync/class-upload-sync.php (hunk 1 above)
Comment (pereirinha) [reply php/sync/class-upload-sync.php:94]:
```suggestion
					'cloudinary-clean-up' => __( 'Clean Cloudinary Error', 'cloudinary' ),
```

File: php/sync/class-upload-sync.php (hunk 2 above)
Comment (pereirinha) [reply php/sync/class-upload-sync.php:164]:
```suggestion
				esc_html__( 'Clean Cloudinary Error', 'cloudinary' )
```

**Comment by pereirinha** [comment]
> There are two very small comments regarding capitalisation. If you think they don't matter then you can ignore!
> 
> I've tested all three scenarios also and they are working.

Thank you @tharsheblows. I took care of these.

**Pull request merged**